akka-http-bench-jmh/src/main/scala/akka/http/impl/engine/http2/H2ServerSwitchBenchmark.scala

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (C) 2020-2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.http.impl.engine.http2
+
+import akka.actor.ActorSystem
+import akka.http.CommonBenchmark
+import akka.http.impl.engine.http2.FrameEvent.HeadersFrame
+import akka.http.impl.engine.http2.framing.FrameRenderer
+import akka.http.scaladsl.model.{ HttpResponse }
+import akka.http.scaladsl.settings.ServerSettings
+import akka.stream.ActorMaterializer
+import akka.stream.TLSProtocol.{ SslTlsInbound, SslTlsOutbound }
+import akka.stream.scaladsl.{ Flow, Keep, Sink, Source }
+import akka.util.ByteString
+import com.typesafe.config.ConfigFactory
+import org.openjdk.jmh.annotations._
+
+import java.util.concurrent.{ CountDownLatch, TimeUnit }
+import scala.concurrent.{ Await, Future }
+import scala.concurrent.duration._
+
+class H2ServerSwitchBenchmark extends CommonBenchmark {
```

Comment:
Can you add a small description what this is about (compared to the other benchmark)?